source/world/entity/Entity.cpp

```diff
@@ -132,7 +132,7 @@ int Entity::move(float x, float y, float z)
 	bool b1, b2, b3, b4, b5, b6;
 	AABB hitold;
 
-	oldX = m_pos.x, oldZ = m_pos.z;
+    oldX = m_pos.x; oldZ = m_pos.z;
```

Comment:
Why change this?

Reply:
Xcode was giving me warnings about it. The changes I made to it had no affect on the runtime of the program. Does it matter that I changed it?